src/resources/postcss/tribe-common-admin/settings/_modal-nav.pcss

```diff
@@ -6,122 +6,124 @@
  *
  * ----------------------------------------------------------------------------- */
 
-.tribe_events_page_tec-events-settings .tribe_settings.wrap {
-	.tec-nav__modal-controls {
-		align-items: center;
-		display: flex;
-		flex-flow: column nowrap;
-		margin: 0;
-		padding: 0 var(--tec-spacer-7) var(--tec-spacer-5);
-
-		@media (--viewport-xsmall) {
-			display: none;
-		}
-
-		.tec-nav__modal-title {
-			font-size: var(--tec-font-size-5);
-			font-weight: var(--tec-font-weight-bold);
-			width: 100%;
-		}
-
-		.tec-modal__control {
+.tribe_events_page_tec-events-settings, .tickets_page_tec-tickets-settings {
+	.tribe_settings.wrap {
+		.tec-nav__modal-controls {
 			align-items: center;
-			background-color: var(--tec-color-background);
-			border: 1px solid var(--tec-admin-core-grey-10);
-			border-radius: var(--tec-border-radius-default);
-			color: var(--tec-admin-core-grey-90);
 			display: flex;
-			font-size: var(--tec-font-size-3);
-			justify-content: space-between;
-			line-height: var(--tec-line-height-2);
-			padding: var(--tec-spacer-1) var(--tec-spacer-4) var(--tec-spacer-1) var(--tec-spacer-2);
-			width: 100%;
-		}
+			flex-flow: column nowrap;
+			margin: 0;
+			padding: 0 var(--tec-spacer-7) var(--tec-spacer-5);
 
-		.tec-modal__control--open .tec-modal__control-icon {
-			height: var(--tec-spacer-2);
-		}
-	}
+			@media (--viewport-xsmall) {
+				display: none;
+			}
+
+			.tec-nav__modal-title {
+				font-size: var(--tec-font-size-5);
+				font-weight: var(--tec-font-weight-bold);
+				width: 100%;
+			}
 
-	#tec-settings-nav-modal {
-		background-color: var(--tec-admin-core-grey-0);
-		padding: var(--tec-spacer-3) 0;
+			.tec-modal__control {
```

Comment:
This looks like there was a lot of moving things around - what actually changed? Rearranging things affects the cascade so do it carefully.